nifi-nar-bundles/nifi-confluent-platform-bundle/nifi-confluent-schema-registry-service/src/main/java/org/apache/nifi/confluent/schemaregistry/ConfluentSchemaRegistry.java

```diff
@@ -52,6 +52,8 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import java.util.regex.Pattern;
+import static org.apache.commons.lang3.StringUtils.trimToEmpty;
```

Comment:
I believe our coding guidelines and checkstyle discourage static imports.

Reply:
I don't think static imports have ever been blocked by checkstyle. Is that something we need to verify as behavior?